### ISSUE TITLE
Prefab Merged Selection Button is Stretched Vertically

### ISSUE THREAD

**Issue by excaliburHisSheath** [FlaShG/GitMerge-for-Unity#40]
![gitmerge_prefabbuttonstretching](https://cloud.githubusercontent.com/assets/1900829/6156470/2ad4bc2e-b200-11e4-99b4-0c72c26bec1f.PNG)

To the right of the spot for choosing the prefab to merge is the little button that brings up a list of all prefabs. The button is stretched vertically, meaning the size of the gui rectangle probably just needs to be tweaked.

**Comment by FlaShG** [comment]
The circle is part of the property field used. The whole field is stretched vertically and as a result, so is the circle.
I have not found a way to change that yet (except to reset the field height to its default).

**Comment by excaliburHisSheath** [comment]
Is there a reason why you wouldn't just want to reset the height to its default? I guess it's nice that the field is a little bit bigger than normal, but I don't know that it's that much of an advantage.

**Comment by FlaShG** [comment]
I just think that a big field to drop stuff into is a bigger advantage than an ugly scaled icon is a disdvantage :grinning:

**Comment by excaliburHisSheath** [comment]
Fair enough. I'll keep an eye out for a way to fix the stretching issue or get a large property field without that icon.

**Comment by FlaShG** [comment]
This has been fixed some time on the way to 2020.3, where it's not stretching anymore. Also, the fix for prefab merging changes this field to a regular Box instead of an ObjectField.
